packages/core/src/connectors/consts.ts

```diff
@@ -1,16 +1,20 @@
-// TODO: move to environment variable
-export const connectorPackages = [
-  '@logto/connector-alipay',
-  '@logto/connector-alipay-native',
-  '@logto/connector-aliyun-dm',
-  '@logto/connector-aliyun-sms',
-  '@logto/connector-apple',
-  '@logto/connector-facebook',
-  '@logto/connector-github',
-  '@logto/connector-google',
-  '@logto/connector-sendgrid-email',
-  '@logto/connector-smtp',
-  '@logto/connector-twilio-sms',
-  '@logto/connector-wechat',
-  '@logto/connector-wechat-native',
-];
+import { getEnv } from '@silverhand/essentials';
+
+export const connectorPackages = getEnv(
```

Comment:
it'll be great if we can provide another option to extend connectors instead of override.

actually shall we extend by default?

Reply:
If we decide to extend the env, then the env should be named as "custom connectors", as the "official" connectors are provided in core source code and cannot be changed without updating the source code.

Reply:
actually the final solution will be using database to store connector info. need revisit later

Reply:
updated with `EXTRA_CONNECTOR_PACKAGES`, PTAL